GoMeet/src/test/java/com/google/sps/VoteMeetingTimeServletTest.java

```diff
@@ -0,0 +1,258 @@
+    when(mockedRequest.getParameter(MeetingTimeFields.VOTERS)).thenReturn(EXISTING_VOTER);
+    new VoteMeetingTimeServlet().doPost(mockedRequest, mockedResponse);
+    
+    // check that error response was sent with appropriate details
+    testBadRequest(HttpServletResponse.SC_CONFLICT, ErrorMessages.USER_HAS_VOTED_ERROR);
+  }
+
+  /** Voter is the first voter - there were no past voters for the meeting time */
+  @Test
+  public void testFirstVoter() throws IOException {
+    // set the voters list in the hardcoded entity to be an empty list
+    Entity meetingTime = getHardcodedEntity();
+    meetingTime.setProperty(MeetingTimeFields.VOTERS, new ArrayList<String>());
+    // set the vote count to be 0
+    meetingTime.setProperty(MeetingTimeFields.VOTE_COUNT, 0);
+    datastore.put(meetingTime);
+
+    when(mockedRequest.getParameter(MeetingTimeFields.MEETING_TIME_ID)).thenReturn(MEETING_TIME_ID_VAL); 
+    when(mockedRequest.getParameter(MeetingTimeFields.VOTERS)).thenReturn(NEW_VOTER);
+    new VoteMeetingTimeServlet().doPost(mockedRequest, mockedResponse);
+
+    // Check that the entity was updated as expected.
+    Entity updatedMeetingTime = getHardcodedEntity();
+
+    // voters should now JUST include NEW_VOTER, and NOT the original voters
+    HashSet<String> voters = new HashSet<String>( (ArrayList<String>) updatedMeetingTime.getProperty(MeetingTimeFields.VOTERS));
+    assertTrue(voters.contains(NEW_VOTER));
+    assertFalse(voters.containsAll(VOTERS_VAL));
+
+    // vote count should be 1
+    int voteCount = ((Long) updatedMeetingTime.getProperty(MeetingTimeFields.VOTE_COUNT)).intValue();
+    assertTrue(voteCount == 1);
+
+    // HttpServletResponse.SC_OK status code should be returned
+    HashMap<String, Object> status = new HashMap<String, Object>() {{
+      put("status", HttpServletResponse.SC_OK);
+    }};
+    writer.flush(); // writer may not have been flushed yet
+    assertNotNull(stringWriter.toString());
+    assertTrue(stringWriter.toString().contains(ServletUtil.convertMapToJson(status)));
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  /**
+   * Retrieved the one hardcoded entity from Local Datastore
+   * @return the hardcoded MeetingTime Entity 
+   */
+  private Entity getHardcodedEntity() {
+    Query query = new Query("MeetingTime");
+    PreparedQuery preparedQuery = datastore.prepare(query);
+    Entity result = preparedQuery.asSingleEntity();
+    return result;
+  }
+
+  /**
+   * Check that the hardcoded MeetingTime Entity in Local Datastore
+   * has the same values as what it was originally hardcoded with.
+   * @return true if the hardcoded entity was not modified, otherwise false
+   */
+  private boolean checkNothingModified() {
+    Entity meetingTime = getHardcodedEntity();
+    // voters should just include the original voters
+    HashSet<String> voters = new HashSet<String>( (ArrayList<String>) meetingTime.getProperty(MeetingTimeFields.VOTERS));
```

Comment:
This line is quite long, could you break down this line? And maybe see if other long lines could be break down as well

Reply:
Yep, I've fixed the line wrapping across all the files in this PR to match the 100 column limit as per the style guide 👍